data/scripts/lib/register_monster_type.lua

```diff
@@ -159,9 +159,12 @@ registerMonsterType.events = function(mtype, mask)
 end
 registerMonsterType.loot = function(mtype, mask)
 	if type(mask.loot) == "table" then
+		local lootError = false
 		for _, loot in pairs(mask.loot) do
 			local parent = Loot()
-			parent:setId(loot.id)
+			if not parent:setId(loot.id) then
+				lootError = true
```

Comment:
Would be a good ide to print out the id / name + you can remove the lootError code and just print here insted
Same for line 187

Reply:
The error handling print is done in the c++ function, so we can differentiate the error.
If I moved the print inside here it would result in double the prints for actually no reason, it doesn't need to remind me after every line that this monster has an error, I'd rather have a summary of errors and at the end it shows which monster it is.